pkg/targetgroupbinding/resource_manager.go

```diff
@@ -22,23 +23,29 @@ type ResourceManager interface {
 }
 
 // NewDefaultResourceManager constructs new defaultResourceManager.
-func NewDefaultResourceManager(k8sClient client.Client, elbv2Client services.ELBV2, logger logr.Logger) *defaultResourceManager {
+func NewDefaultResourceManager(k8sClient client.Client, elbv2Client services.ELBV2,
```

Comment:
Did you modify the main.go as well? if so please include it together.

Reply:
update the PR to modify main.go